cmd/src/api.go

```diff
@@ -134,11 +155,16 @@ func curlCmd(endpoint, accessToken, query string, vars map[string]interface{}) (
 	}
 
 	s := fmt.Sprintf("curl \\\n")
-	if accessToken != "" {
-		s += fmt.Sprintf("   %s \\\n", shellquote.Join("-H", "Authorization: token "+accessToken))
+	if cfg.AccessToken != "" {
+		headerName, headerValue := authenticationHeader(cfg)
+		if headerName == "" {
+			return "", fmt.Errorf(authHeaderMissingAccessTokenError)
```

Comment:
I don't think returning this error is suitable here. This is a bug in our application so we should be panicking here.

```Go
panic("failed to create authentication header from configuration")
```